### DISCUSSION TITLE
How I can delete my teleport.sh account?

### DISCUSSION THREAD

**Discussion by Huhusski212121** [gravitational/teleport#32265] [Q&A] [answered]
How I can delete my teleport.sh account?

**Comment by webvictim** [comment, marked as answer]
If you have a 14-day Teleport Team trial, it will be automatically deleted at the end of your trial period.

If you have a Teleport Enterprise Cloud account, please reach out to sales@goteleport.com instead.

**Reply by Huhusski212121** [reply]
what you mean with automatically deleted , I will not can login to my account or when I login the website asking me for buy teleport enterprise ?

**Reply by scottpgallagher** [reply]
It will be terminated (no longer accessable) after the 14 days if you don't input pay information.

**Comment by molinalucas** [comment]
How can I delete my accounts immediately qithout pay for another month????

Jesus Christ

**Reply by molinalucas** [reply]
Please, how can I destroy it for the whole ever? It will bankrupt me

**Reply by webvictim** [reply]
Assuming you're using Teleport Team, you should be able to cancel your subscription from inside your Teleport dashboard.

If you're having trouble doing that, please email support at goteleport dot com with details and we'll assist you as soon as possible.

**Reply by molinalucas** [reply]
I could do so, my doubt is about what exactly I am going to pay 19th December.

Would it be another 40 bucks?
![image](https://github.com/gravitational/teleport/assets/48143182/60324cd9-4abe-49f5-b611-3f2d2b1091c4)

**Reply by molinalucas** [reply]
Is there a email a channel that support paid customers that not pay for enterprise?

**Reply by molinalucas** [reply]
support@goteleport.com 

I will try it, hope that is better. 

I can't offer your products for any aws customers, they would not accept say that happers, and here, on github, with lack of total care for support. 

Ges. 

There are at least 2 products that make what teleport makes for free. And the paid plans is for some support. 

Oh boy.

**Reply by molinalucas** [reply]
I sent this github discussion to support, just a great idea.
